src/wifi.rs

```diff
@@ -336,10 +355,11 @@ impl<'d> WifiDriver<'d> {
     ) -> Result<Self, EspError> {
         Self::init(nvs.is_some())?;
 
-        let (status, subscription) = Self::subscribe(&sysloop)?;
+        let (status, subscription) = Self::subscribe_statuses(&sysloop)?;
```

Comment:
Nit but... if we don't change `WifiDriver` because of my previous comment, let's not rename anything for now.